### PR TITLE
Run black on the entire repo

### PR DESCRIPTION
**Pull request by markusdregi**
This PR formats the entire repo using black.

The first commit contains the reformatting, the second makes the style GitHub workflow check the entire repo and the last removes the `pytest` driven style test. The later two are opinionated and open for discussion :)

### CODE REVIEW

**Comment by sondreso** [comment]
🚀  Add [Code style: black](https://github.com/psf/black) to `README`?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ert [![Build Status](https://travis-ci.org/equinor/ert.svg?branch=master)](https://travis-ci.org/equinor/ert)
+# ert [![Build Status](https://travis-ci.org/equinor/ert.svg?branch=master)](https://travis-ci.org/equinor/ert)[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (sondreso) [inline README.md:1]: Very nitpick, but maybe include a space?
```suggestion
# ert [![Build Status](https://travis-ci.org/equinor/ert.svg?branch=master)](https://travis-ci.org/equinor/ert) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

Comment (markusdregi) [reply README.md:1]: If there is a PR that should accept style comments, it better be this one 😂 I was just about to check how it rendered... Also, the `travis` badge is outdated, so I'm creating a third PR 🚢

**Review by sondreso — APPROVED** [review]
🖤

**Pull request merged**